csharp/Platform.Collections/Segments/Walkers/AllSegmentsWalkerBase[T, TSegment].cs

```diff
@@ -96,7 +89,7 @@ public virtual void WalkAll(IList<T> elements)
         /// <para>
         /// Iterations the segment.
         /// </para>
-        /// <para></para>
+        /// <para>Представляет базоый метод, для всех методов итерации.</para>
```

Comment:
This documentation is incorrect. Check any documentation of abstract classes: [Stream abstract class](https://docs.microsoft.com/en-us/dotnet/api/system.io.stream?view=net-5.0) and its methods